sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/model/NameCityStateId.java

```diff
@@ -87,6 +88,27 @@ public NameCityStateId(String name, String city, String state, long id) {
     this.id = id;
   }
 
+  @Override
+  public boolean equals(Object otherObject) {
+    if (this == otherObject) {
+      return true;
+    }
+    if (otherObject == null || getClass() != otherObject.getClass()) {
+      return false;
+    }
+
+    NameCityStateId other = (NameCityStateId) otherObject;
+    return name.equals(other.name)
```

Comment:
Unless we have [`@DefaultAnnotation(NonNull.class)`](https://github.com/apache/beam/blob/645e1b51e1c83ec4b0b54a774d6615624f3c42ed/sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/transform/agg/package-info.java#L22)  in `package-info.java` for this package, then I think we should use `Objects.equals()` for equality checks

Reply:
Fixed. Using Objects.equals() everywhere now.